### PR TITLE
haskell-org-committee: merge in content from the wiki

### PR DESCRIPTION
**Pull request by ketzacoatl**
As outlined in #59, there are two pages which describe the Haskell.org committee and it's roles, responsibilities, process, history, etc. One is here on haskell.org, the other is on the wiki at https://wiki.haskell.org/Haskell.org_committee. Both pages appear authoritative, but differ in their content. At the same time, the general consensus is to move content off the wiki (particularly the more "official" content such as this). We would then replace that wiki article with a link to the this page. This is a first attempt which is more of a "bulk import" (or "merge") of the existing content. Note that there are a few places where the differences in content on the haskell.org page and wiki article may seem to be in conflict (or where one does not go as far as the other). These discrepancies and conflicts should be reviewed and updated. Some of the wiki content is more verbose than that on this page, and may still deserve some word-smithing.

A rough list of changes:
* collected the full list of links at the bottom of the page
* move the list of "committee members" a little lower in the page, bumping contact info above
* Included/merged wiki content: process, history and responsibilities content

I can make additional updates to this, but would need feedback on that. cc @tikhonjelvis

### CODE REVIEW

File: haskell-org-committee.markdown [hunk 1]
```diff
@@ -14,15 +14,21 @@ Haskell.org is a 501(c)(3) non-profit organization that oversees the Haskell.org
   * [Discourse][discourse]
   * [Hoogle], a type-aware Haskell search engine
 
-Haskell.org also organizes Haskell's participation in  the [Google Summer of Code program][summer].
+Its purpose is to promote educational and scientific progress relating to the Haskell programming language and related technologies.
```
Comment (ketzacoatl) [inline haskell-org-committee.markdown:17]: I think this "scientific progress" could go, but I wanted to leave it as-is on the first commit, and I'm not sure what to replace it with (and then once its gone, the sentence needs more work).

File: haskell-org-committee.markdown [hunk 2]
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
   * Rebecca Skinner (term ends 2022)
   * Alexandre Garcia de Oliveira (term ends 2022)
 
-In November of each year—or if a member steps down before their term is over—the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
```
Comment (ketzacoatl) [inline haskell-org-committee.markdown:47]: This bit was moved over to the "membership process" section.

File: haskell-org-committee.markdown [hunk 3]
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
...
-
 Committee members do not need to be deep technical Haskell experts.  Instead we look for enthusiasm towards improving the Haskell community.  We aim to represent the different facets of the community.  We aim to be diverse in terms of industry or research, and in terms of gender, race and location.
 
+### Process
+
+* The committee's governing rules are laid out in the [Board Resolution](https://wiki.haskell.org/wikiupload/9/9c/Haskell-org-board-res.pdf) of that month.
```
Comment (ketzacoatl) [inline haskell-org-committee.markdown]: I think this PDF should be imported/added here to the site, and hosted here / not lost.

File: haskell-org-committee.markdown [hunk 4]
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
   * Rebecca Skinner (term ends 2022)
   * Alexandre Garcia de Oliveira (term ends 2022)
```
Comment (ketzacoatl) [inline haskell-org-committee.markdown:50]: I skipped making updates to this list (as the dates point out an update is needed), but can apply them and that would address #62 (or that list can be updated in a separate PR).

Comment (ketzacoatl) [reply haskell-org-committee.markdown:50]: Scratch that, as Tikhon notes in that ticket, the vote was delayed and an update is pending (there's no update to make right now, so ignore this thread).

File: haskell-org-committee.markdown (hunk 1 above)
Comment (TikhonJelvis) [reply haskell-org-committee.markdown:17]: I'm pretty sure this is just quoting our mission statement from our 501(c)(3) documents—as I understand it, this is stock language educational nonprofits use to describe why they can be a nonprofit.

File: haskell-org-committee.markdown (hunk 3 above)
Comment (TikhonJelvis) [reply haskell-org-committee.markdown]: Good call. I figure we can just add it to the repo.

If you're not sure where to put it so that it's available as a download, I can give it a try—I don't know myself off-hand, so I'd have to poke around the code a bit.

File: haskell-org-committee.markdown [hunk 5]
```diff
@@ -14,15 +14,21 @@ Haskell.org is a 501(c)(3) non-profit organization that oversees the Haskell.org
...
+Some particular aspects of the work of Haskell.org include producing and maintaining material regarding the Haskell language on haskell.org and other associated websites; hosting and maintaining shared assets and infrastructure for the benefit of the Haskell community; coordinating educational activities including mentorship programs; and potentially organizing event regarding the Haskell language.
+
+Haskell.org also organizes Haskell's participation in  the [Google Summer of Code program][gsoc].
+
+### Contact
+
+  * [committee@haskell.org][email]: directly email the committee
+  * [haskell-community mailing list][list]: public discussions about the committee and Haskell community infrastructure
+  * [GitHub][github]: issues, suggestions and pull requests for the website
+  * TODO: add one for proposal process?
```
Comment (TikhonJelvis) [inline haskell-org-committee.markdown]: Adding a link to the proposals repo is a great idea. I guess we should also write up a description of the process in that repo's README or something...

File: haskell-org-committee.markdown [hunk 6]
```diff
@@ -14,15 +14,21 @@ Haskell.org is a 501(c)(3) non-profit organization that oversees the Haskell.org
   * [Discourse][discourse]
   * [Hoogle], a type-aware Haskell search engine
 
-Haskell.org also organizes Haskell's participation in  the [Google Summer of Code program][summer].
+Its purpose is to promote educational and scientific progress relating to the Haskell programming language and related technologies.
+
+As a whole it seeks to service the open source Haskell community.
```
Comment (TikhonJelvis) [inline haskell-org-committee.markdown]: Seems like we can just drop this sentence—it overlaps with the previous paragraph and doesn't add any real information.

File: haskell-org-committee.markdown [hunk 7]
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
...
   * Alexandre Garcia de Oliveira (term ends 2022)
 
-In November of each year—or if a member steps down before their term is over—the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
-
 Committee members do not need to be deep technical Haskell experts.  Instead we look for enthusiasm towards improving the Haskell community.  We aim to represent the different facets of the community.  We aim to be diverse in terms of industry or research, and in terms of gender, race and location.
 
+### Process
+
+* The committee's governing rules are laid out in the [Board Resolution](https://wiki.haskell.org/wikiupload/9/9c/Haskell-org-board-res.pdf) of that month.
+* The committee uses the [haskell-community][list] for public discussion, where all interested parties may participate.
```
Comment (TikhonJelvis) [inline haskell-org-committee.markdown]:
```suggestion
* The committee uses the [haskell-community mailing list][list] for public discussion, where all interested parties may participate.
```

File: haskell-org-committee.markdown (hunk 5 above)
Comment (TikhonJelvis) [inline haskell-org-committee.markdown]:
```suggestion
  * [Haskell.org proposals][repo]: submit community proposals for Haskell.org
```

File: haskell-org-committee.markdown [hunk 8]
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
...
+* The committee will seek to reach consensus among its membership on its decisions. Ultimately, the committee will make decisions by more than half of the membership voting for a particular outcome.
+* These rules of operation may also be changed in the same way.
+
+#### Committee Membership
+
+The committee consists of 7 members. Members are expected to serve a 3 year term, and terms are staggered so that 2 or 3 members step down each year, at the end of October.
+
+The members will elect one of their number to be chair each year. The chair is responsible for making sure that things keep moving, and to ensure that a conclusion is reached on any issues raised.
+
+In November of each year — or if a member steps down before their term is over — the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
```
Comment (TikhonJelvis) [inline haskell-org-committee.markdown]:
```suggestion
In November of each year—or if a member steps down before their term is over—the committee holds an open call for new members, encouraging self-nominations. Committee members whose terms are over are eligible for renomination.
```

For consistency with other em-dashes (and good typography in general) :).

File: haskell-org-committee.markdown [hunk 9]
```diff
@@ -44,25 +50,47 @@ The committee consists of 7 members serving 3-year terms. The current members ar
...
+
 After nominations have been solicited, the current committee—including any outgoing members—selects new members with a simple majority vote.
```
Comment (TikhonJelvis) [inline haskell-org-committee.markdown]:
```suggestion
After nominations have been solicited, the current committee—including any outgoing members not seeking reelection—selects new members with a simple majority vote.
```

The Wiki page is right: any members who renominated themselves after their term is up don't vote that year. The current page is missing that detail.

**Review by TikhonJelvis — COMMENTED** [review]
Thanks for putting this together! I left a few suggestions.

Also, could you point this PR back to the [haskell-infra] repo? It's currently set to merge into the `master` branch of your fork instead.

Thanks!

[haskell-infra]: https://github.com/haskell-infra/www.haskell.org

File: haskell-org-committee.markdown (hunk 1 above)
Comment (ketzacoatl) [reply haskell-org-committee.markdown:17]: OK, I guess the statement should remain as-is?

File: haskell-org-committee.markdown (hunk 6 above)
Comment (ketzacoatl) [reply haskell-org-committee.markdown]: OK, I will update that.

**Comment by ketzacoatl** [comment]
> Thanks for putting this together! I left a few suggestions.

Thank you for reviewing and helping me get it done :)


> Also, could you point this PR back to the [haskell-infra](https://github.com/haskell-infra/www.haskell.org) repo? It's currently set to merge into the `master` branch of your fork instead.

Yes, sorry, I overlooked that before. And it looks like I'll need to create a new PR to target the other remote.

**Comment by TikhonJelvis** [comment]
Sounds good. Shame GitHub doesn't make it easy to point a PR to a different repo :/.

I'm going to be super busy for a few days, but if you set up a new PR, I should be able to review it early next week. After that it'll need one more approval from somebody on the committee—we have a two-approval process for website changes—and it should be ready to go.

Thanks!

**Comment by ketzacoatl** [comment]
OK, thank you for the heads up on all of that. I will probably pick this back up on the weekend, and will aim to sync up again next week.

File: haskell-org-committee.markdown (hunk 6 above)
Comment (ketzacoatl) [reply haskell-org-committee.markdown]:
```suggestion
```

File: haskell-org-committee.markdown (hunk 5 above)
Comment (ketzacoatl) [reply haskell-org-committee.markdown]: @TikhonJelvis, sorry for being a derp, but where is that proposals repo? I've been unable to locate it.

File: haskell-org-committee.markdown (hunk 3 above)
Comment (ketzacoatl) [reply haskell-org-committee.markdown]: I've gotten that worked out, I think.

File: haskell-org-committee.markdown (hunk 5 above)
Comment (ketzacoatl) [reply haskell-org-committee.markdown]: Oh, I found it referenced in another PR - https://github.com/haskell-org/committee. I'll add that now.

Comment (ketzacoatl) [reply haskell-org-committee.markdown]: oh, this is already added, resolving.

**Comment by ketzacoatl** [comment]
This PR has been replaced by https://github.com/haskell-infra/www.haskell.org/pull/64.
